### ISSUE TITLE
Documentation: How to get started

### ISSUE THREAD

**Issue by paschelino** [EventStore/EventStore.JVM#76]
should contain
- sample command line code how check for eventstore service availability
- mvn repository dependency declaration example
- some example code to put into an integration test how to create streams and how to write and read to them
- links to more advanced topics

**Comment by gregoryyoung** [comment]
"sample command line code how check for eventstore service availability"

Would depend entirely on your configuration default configuration http://127.0.0.1:2113 (shows admin console) or telnet 127.0.0.1 1113. But these only work on default configuration

isn't 3 on the readme of this repository already or do you mean something else?

**Comment by paschelino** [comment]
Hi Gregory,

thank you for your quick reply! Sorry for the incomprehensible description I provided to that ticket. My head was a bit tired after work. I'm thinking from the newbies perspective and wanted to suggest, it would be helpful to provide a _getting started_ thing, which is a somehow failsafe way to set up some initial code doing basic stuff on an event stream successfully. This could include some checkpoints during the process, which show to someone using the lib, that a 'milestone' in the setup is done ;-) .  

1. checkpoint: a command line call similar to a ping or a --version call as a proof, eventstore is up and running. One could also create a first event via cURL to check, but could fail due to unrelated errors having to do with the event creation itself. Then you would need to find the error of your request or your installation...
3. yes. There is sample code for that. But I think for a getting started, it is not too useful because the code snippets try to show too many things at once. An integration test would be very helpful, since connection setup code and actual writing / reading events / streams code would be separated in a natural way and the parts of such a test could focus on a very particular topic each. And last but not least this is easily executable via one's IDE and leads to a nice encouraging success at last: A green test. :-)

**Comment by paschelino** [comment]
A separate sample project on github with a series of tests for the newbie to make them green one by one could also be an idea.

**Comment by michielboekhoff** [comment]
@paschelino I agree that documentation is a weak point of the JVM client. I've recently been contributing to a new component and I've pondered writing some documentation for this project myself. What exactly are you looking for? I was thinking the API reference should really be expanded upon, too.
